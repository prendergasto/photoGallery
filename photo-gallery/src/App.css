.App {
  text-align: center;
}

ol ul {
  display: flex;
  justify-content: space-evenly;
}

img {
  height: 200px;
  width: 200px;
}

