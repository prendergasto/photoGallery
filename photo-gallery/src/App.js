import React, { Component } from 'react';
import Header from "./header.js";
import ImageItem from "./imageItem.js"
import ImageList from "./imageList.js"
// import logo from './logo.svg';
import images from './data.js';
import './App.css';


export default class App extends Component {
  render() {
     return (
      <div id="container">
        <Header />
        <ImageList images={images}/>
    </div>
      
    )
  }

};