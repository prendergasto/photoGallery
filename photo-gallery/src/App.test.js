import React from 'react';
// import { render } from '@testing-library/react';
import App from './App';
import renderer from "react-test-renderer";
test('renders App.js correctly', () => {
  const tree = renderer
  .create(<App />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});