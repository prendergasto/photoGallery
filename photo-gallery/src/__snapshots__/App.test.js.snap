// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.js correctly 1`] = `
<div
  id="container"
>
  <div>
    <h1>
      Choose something weird...
    </h1>
  </div>
  <div
    id="list-container"
  >
    <ol>
      <ul>
        <img
          alt=""
          src="http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg"
        />
        <h2>
          UniWhal
        </h2>
        <p
          className="dataDescription"
        >
          A unicorn and a narwhal nuzzling their horns
        </p>
        <h3>
          Keyword: 
          narwhal
        </h3>
        <h3>
          # of horns: 
          1
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://images.unsplash.com/photo-1512636618879-bbe79107e9e3?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=bd9460ee6d1ddbb6b1ca7be86dfc4590&auto=format&fit=crop&w=1825&q=80"
        />
        <h2>
          Rhino Family
        </h2>
        <p
          className="dataDescription"
        >
          Mother (or father) rhino with two babies
        </p>
        <h3>
          Keyword: 
          rhino
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://www.dhresource.com/0x0s/f2-albu-g5-M00-1A-11-rBVaI1hsIIiALxKzAAIHjSU3VkE490.jpg/wholesale-halloween-costume-prop-unicorn.jpg"
        />
        <h2>
          Unicorn Head
        </h2>
        <p
          className="dataDescription"
        >
          Someone wearing a creepy unicorn head mask
        </p>
        <h3>
          Keyword: 
          unicorn
        </h3>
        <h3>
          # of horns: 
          1
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://images.unsplash.com/photo-1518946222227-364f22132616?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=4836a6fca62e7dce9324346bacfde085&auto=format&fit=crop&w=2534&q=80"
        />
        <h2>
          UniLego
        </h2>
        <p
          className="dataDescription"
        >
          Lego figurine dressed in a unicorn outfit
        </p>
        <h3>
          Keyword: 
          unilego
        </h3>
        <h3>
          # of horns: 
          1
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://i.pinimg.com/736x/b4/61/06/b46106830b841017ea59870b27ec18dc--narwhals-a-unicorn.jpg"
        />
        <h2>
          Basically a unicorn
        </h2>
        <p
          className="dataDescription"
        >
          A narwhal is basically a unicorn after all, right?
        </p>
        <h3>
          Keyword: 
          narwhal
        </h3>
        <h3>
          # of horns: 
          1
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://i.pinimg.com/originals/16/cf/2a/16cf2a0b3fd51b9bee08bb6296193b75.jpg"
        />
        <h2>
          #truth
        </h2>
        <p
          className="dataDescription"
        >
          The truth behind narwhals
        </p>
        <h3>
          Keyword: 
          narwhal
        </h3>
        <h3>
          # of horns: 
          1
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://secure.img1-ag.wfcdn.com/im/17007094/resize-h800%5Ecompr-r85/3589/35892451/Baby+Rhino+Figurine.jpg"
        />
        <h2>
          Baby Rhino
        </h2>
        <p
          className="dataDescription"
        >
          This is actually a figurine but it looks kinda real
        </p>
        <h3>
          Keyword: 
          rhino
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://vignette.wikia.nocookie.net/landbeforetime/images/c/c3/Cera_infobox.png/revision/latest?cb=20180422005003"
        />
        <h2>
          Cera
        </h2>
        <p
          className="dataDescription"
        >
          Three horns but still, horns. And who doesn't like The Land Before Time?
        </p>
        <h3>
          Keyword: 
          triceratops
        </h3>
        <h3>
          # of horns: 
          3
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://ae01.alicdn.com/kf/HTB18GwSQVXXXXaZaXXXq6xXFXXXh/Animal-Cosplay-Costume-Narwhal-Onesie-Mens-Womens-Cartoon-Whale-Pajamas.jpg"
        />
        <h2>
          Narwhal costume
        </h2>
        <p
          className="dataDescription"
        >
          A woman wearing a blue narwhal costume
        </p>
        <h3>
          Keyword: 
          narwhal
        </h3>
        <h3>
          # of horns: 
          1
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://www.shopmascot.com/image/cache/mascotnew/new196-800x800.jpg"
        />
        <h2>
          Rhino costume
        </h2>
        <p
          className="dataDescription"
        >
          Mascots have to get their costumes somewhere
        </p>
        <h3>
          Keyword: 
          rhino
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://www.tinselbox.com/wp-content/uploads/2018/03/I-BELIEVE-IN-UNICORNS-FREE-PRINTABLE-WATERCOLOR-7-of-1.jpg"
        />
        <h2>
          Believe
        </h2>
        <p
          className="dataDescription"
        >
          I believe in unicorns, do you?
        </p>
        <h3>
          Keyword: 
          unicorn
        </h3>
        <h3>
          # of horns: 
          1
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Markhor_Schraubenziege_Capra_falconeri_Zoo_Augsburg-02.jpg/220px-Markhor_Schraubenziege_Capra_falconeri_Zoo_Augsburg-02.jpg"
        />
        <h2>
          Markhor
        </h2>
        <p
          className="dataDescription"
        >
          These wild goats eat snakes, then secrete a foam that locals fight over for the antivenom properties -- how cool is that?
        </p>
        <h3>
          Keyword: 
          markhor
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="http://www.zooborns.com/.a/6a010535647bf3970b0223c84d5959200c-800wi"
        />
        <h2>
          Baby markhor
        </h2>
        <p
          className="dataDescription"
        >
          Even the babies are adorable
        </p>
        <h3>
          Keyword: 
          markhor
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://cdn.modernfarmer.com/wp-content/uploads/2014/08/28476658_9c97f35096_o.jpg"
        />
        <h2>
          Mouflon
        </h2>
        <p
          className="dataDescription"
        >
          Those horns though
        </p>
        <h3>
          Keyword: 
          mouflon
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://cdn.modernfarmer.com/wp-content/uploads/2014/08/addax.jpg"
        />
        <h2>
          Addax
        </h2>
        <p
          className="dataDescription"
        >
          This guy is basically extinct but survives well in captivity, so they're frequently found in zoos
        </p>
        <h3>
          Keyword: 
          addax
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://cbsnews3.cbsistatic.com/hub/i/r/2013/03/05/5b414225-a645-11e2-a3f0-029118418759/thumbnail/620x350/2d4cf24685b45c22912e64d2004fec8d/Baby_Mouflon_Wild_Sheep.jpg"
        />
        <h2>
          Baby mouflon
        </h2>
        <p
          className="dataDescription"
        >
          The cuteness that is a baby mouflon asleep
        </p>
        <h3>
          Keyword: 
          mouflon
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://cdn.modernfarmer.com/wp-content/uploads/2014/08/chameleon.jpg"
        />
        <h2>
          Happy Jackson's Chameleon
        </h2>
        <p
          className="dataDescription"
        >
          These are really common in Hawaii
        </p>
        <h3>
          Keyword: 
          chameleon
        </h3>
        <h3>
          # of horns: 
          2
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://imgc.allpostersimages.com/img/print/posters/dlillc-jackson-s-chameleon_a-G-13448768-14258384.jpg"
        />
        <h2>
          Serious Jackson's Chameleon
        </h2>
        <p
          className="dataDescription"
        >
          This one is very serious.
        </p>
        <h3>
          Keyword: 
          chameleon
        </h3>
        <h3>
          # of horns: 
          3
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://www.nps.gov/band/learn/nature/images/short-horned-lizard-open-mouth-18.jpg?maxwidth=650&autorotate=false"
        />
        <h2>
          Horned Lizard
        </h2>
        <p
          className="dataDescription"
        >
          Fave food: ants
        </p>
        <h3>
          Keyword: 
          lizard
        </h3>
        <h3>
          # of horns: 
          100
        </h3>
      </ul>
      <ul>
        <img
          alt=""
          src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/df/Smaug_par_David_Demaret.jpg/290px-Smaug_par_David_Demaret.jpg"
        />
        <h2>
          Smaug
        </h2>
        <p
          className="dataDescription"
        >
          Fan illustration of Smaug from 'The Hobbit'
        </p>
        <h3>
          Keyword: 
          dragon
        </h3>
        <h3>
          # of horns: 
          100
        </h3>
      </ul>
    </ol>
  </div>
</div>
`;
